pkg/ddl/backfilling_dispatcher.go

```diff
@@ -309,8 +293,11 @@ func generateNonPartitionPlan(d *ddl, tblInfo *model.TableInfo, job *model.Job)
 		return nil, err
 	}
 
-	subTaskMetas := make([][]byte, 0, 100)
-	regionBatch := 20
+	// Make subtask large enough to reduce the overhead of local/global flush.
+	quota := variable.DDLDiskQuota.Load()
```

Comment:
it is not a good value for global sort

Reply:
Does the partition table have a similar problem? For now, each partition is a subtask.

Reply:
https://github.com/pingcap/tidb/pull/47982/commits/4e089b984a4547a3926e4680d8bd21f14b6ddfc8